spin1_api/spin1_api.c

```diff
@@ -773,18 +774,19 @@ static void report_warns(void)
 }
 
 
+#ifdef __GNUC__
+    register uint _lr asm("lr");
+#else
+    register uint _lr __asm("lr");
```

Comment:
Change from just above applies here too.

Reply:
As above; we could split the declaration but probably not worth it...